crypto/modes/ccm128.c

```diff
@@ -11,6 +11,12 @@
 #include "modes_lcl.h"
 #include <string.h>
 
+#ifdef __GNUC__
```

Comment:
I'd like to suggest to add #if defined(STRICT_ALIGNMENT) around this. The type is used only in STRICT_ALIGNMENT segments below and the said #if would serve as double-insurance that it's not used outside.

Reply:
yes.

Reply:
Actually it is only used in non-strict-aligmnment section.

Reply:
> Actually it is only used in non-strict-aligmnment section.

I did write "the type is used only in STRICT_ALIGNMENT segments", didn't I? Point behind suggestion is to make it more clear at declaration point. It's readability thing. [Besides double-insurance that is.]

Reply:
Please look again where the type is actually used.